js/projectsConstructor.js

```diff
@@ -9,22 +9,48 @@ function Project (projectsListObj) {
   Project.all.push(this);
 }
 
-Project.prototype.renderProject = function() {
+Project.prototype.toHtml = function() {
   var $newProject = $('.template').clone();
   $newProject.removeClass('template');
   $newProject.find('h3')
-             .text(this.name);
+             .text(this.title);
   $newProject.find('a')
-             .eq(1).attr('href', 'this.url');
+             .eq(0).attr('href', this.url);
   $newProject.find('a')
-             .eq(2).attr('href','this.githubURL');
+             .eq(1).attr('href',this.githubURL);
   $newProject.find('img')
-             .eq(2).src('this.img');
+             .attr('src',this.img);
   $newProject.find('p')
-             .eq(1).text('this.skillsUsed');
+             .eq(0).text(this.skillsUsed);
+  $('.anchorProjects').append($newProject);
+};
 
-  $('.template').append($newProject);
+Project.all = [];
 
-}
+projectsList.forEach(function(project) {
+  new Project(project);
+});
 
-Project.all = [];
+Project.all.forEach(function(project) {
+  project.toHtml();
+});
+
+$('#aboutMe').on('click', function(){
+  $('main').hide();
+  $('main [id|="'+$(this).attr('data-content')+'"]').fadeIn();
```

Comment:
Remember that the way that you select with an id is `#the-id`, so you do not need such a complicated selector here.